assets/src/edit-story/components/carousel/zoomSelector/zoomSelector.js

```diff
@@ -18,20 +18,40 @@
  * External dependencies
  */
 import { useCallback, useMemo } from 'react';
+import styled from 'styled-components';
 import { __, _x } from '@web-stories-wp/i18n';
 /**
  * Internal dependencies
  */
 import { ZOOM_SETTING } from '../../../constants';
 import { useLayout } from '../../../app/layout';
-import { DropDown } from '../../form';
-import { Placement } from '../../popup';
+import { DropDown, PLACEMENT } from '../../../../design-system';
+
+const StyledDropDown = styled(DropDown)`
+  margin-right: 8px;
+
+  button {
+    height: 32px;
+    padding: 8px;
+    padding-right: 0;
+
+    span {
+      padding: 0;
+    }
+  }
```

Comment:
if you want - I added `selectButtonStylesOverride` as a prop in `DropDown` that allows CSS to get injected into the `Select` button so you don't have to dig into the dropdown like this if you don't want to.

Reply:
Oh that's dope, I'll use that instead